controllers/testdata/charts/helmchartwithdeps/Chart.yaml

```diff
@@ -31,3 +31,6 @@ dependencies:
   - name: grafana
     version: ">=5.7.0"
     repository: "https://grafana.github.io/helm-charts"
+  - name: podinfo
+    version: ">=6.1.*"
```

Comment:
It may not be true for how we are using this test data today, but I feel test data with such version could result in unexpected failures when there's a new version in the upstream.
Can we pin this?

Reply:
But it may be true for the grafana dependency too `>=5.7.0`.